### DISCUSSION TITLE
2006/06/26/%E5%A6%82%E4%BD%95%E5%88%9B%E5%BB%BA%E5%92%8C%E4%BD%BF%E7%94%A8-labview-%E4%B8%AD%E7%9A%84-llb-%E6%96%87%E4%BB%B6

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#87] [Announcements]
# 2006/06/26/%E5%A6%82%E4%BD%95%E5%88%9B%E5%BB%BA%E5%92%8C%E4%BD%BF%E7%94%A8-labview-%E4%B8%AD%E7%9A%84-llb-%E6%96%87%E4%BB%B6

最近接连有人问我，怎样在 LabVIEW 中创建一个 LLB 文件。于是我就把它写了下来。 通常，需要新建一个文件时，我们很自然就会想到去选中菜单“File->New”。可是，在 LabVIEW 8 中， 在“新建”菜单中是看不到 LLB 这种文件类型的； 要创建或者管理一个 LLB 文件，首先要选择“Tool->LLB Manager...”。在打开 LLB Manager 之后，再在菜单中选择“File -> New LLB”。这样才能创建一个新的 LLB 文件出来。

https://blog.qizhen.xyz/2006/06/26/%E5%A6%82%E4%BD%95%E5%88%9B%E5%BB%BA%E5%92%8C%E4%BD%BF%E7%94%A8-labview-%E4%B8%AD%E7%9A%84-llb-%E6%96%87%E4%BB%B6

**Comment by ruanqizhen** [comment]
之前博客下的留言：


Loren
2007年06月8日 20:25 编辑
大侠，我是LV新人，newman啊！哈哈，会常来看看，希望以后能交流交流！！

回复
aragorn
2006年11月13日 10:25 编辑
解釋的很清楚耶
謝謝分享

回复
Unknown
2006年08月7日 21:40 编辑
不说我还不知道有llb文件呢，呵呵

回复
永青
2006年06月27日 09:18 编辑
我问了一个好问题啊，劳烦阮大侠下次再解释一下project和library吧，，嗯，下次写本Effective Labview

回复
Jing
2006年06月26日 19:00 编辑
好的。

回复
